hack/log-events.py

```diff
@@ -0,0 +1,56 @@
+import json
+import re
+import sys
+from datetime import datetime
+from enum import Enum
+
+class ActionType(Enum):
+    PUSH = "PUSH"
+    PULL = "PULL"
+
+def process_log(log_message, push_pull):
+    try:
+        # Remove escape characters
+        log_message = log_message.replace('\\"', '"')
+
+        # Extract JSON part from the log message
+        json_part = re.search(r'{.*}', log_message).group()
```

Comment:
you should check if this  return result

Reply:
json_match = re.search(r'{.*}', log_message)
        if not json_match:
            print("No JSON part found in log message.")
            return

Reply:
updated